WalletWasabi/WabiSabi/Backend/Rounds/RoundParameters.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace WalletWasabi.WabiSabi.Backend.Rounds;
 
-public record RoundParameters
+public record RoundParameters : IUtxoSelectionParameters
```

Comment:
Please try hard to avoid adding interfaces (we almost never use them). If you want you can add a new `UtxoSelectionParameters` containing the required properties and that can be build from a round parameter.

Reply:
Done https://github.com/zkSNACKs/WalletWasabi/pull/9123/commits/64f47443ab90c6dd647851fe8d16f85024606ef4.